velox/type/Variant.cpp

```diff
@@ -577,19 +578,45 @@ uint64_t variant::hash() const {
   }
 }
 
-/*static*/ bool variant::equalsFloatingPointWithEpsilon(
-    const variant& a,
-    const variant& b) {
+namespace {
+
+// Compare floating point numbers using relative epsilon comparison.
+// See
+// https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/
+// for details.
+template <TypeKind KIND, typename TFloat, typename TInteger>
```

Comment:
where do you use TInteger?

Reply:
Nice catch. It is not used. Let me remove it.